bims/migrations/0306_auto_20220428_0506.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.2.16 on 2022-04-27 14:19
+
+from django.db import migrations, models
+import django.db.models.deletion
+from sass.enums.chem_unit import ChemUnit
+
+
+def init_unit(apps, schema_editor):
+    """ Initialize table Unit"""
+    Unit = apps.get_model('bims', 'Unit')
+    for unit in ChemUnit:
+        obj = Unit(unit=unit.value)
+        obj.save()
+
+
+def insert_value_to_unit_foreign_key(apps, schema_editor):
+    """ Insert value to field unit"""
+
+    Chem = apps.get_model('bims', 'Chem')
+    Unit = apps.get_model('bims', 'Unit')
+    for unit in ChemUnit:
+        chem_unit = Chem.objects.filter(chem_unit=unit.value)
+        chem_unit.update(unit=Unit.objects.get(unit=unit.value).id)
```

Comment:
@dimasciput please can you review this? In my local I got only 5 chem units update and the rest got null value.